build/Publish.targets

```diff
@@ -43,7 +43,11 @@
       <ForPublishing Include="%(GenerateArchivesInputsOutputs.Outputs)"
                      Condition=" '$(IsLinuxDistroSpecific)' != 'true' "/>
       <ForPublishing Include="$(PackagesDirectory)/Microsoft*.nupkg"
```

Comment:
@wli3 can you review this part of the merge?

Reply:
@johnbeisner do we need this change in master? I think it is about the publish nuget time out, but you fixed it in master

Reply:
Yes, we should have this in master - 2.0.0 is correct. We wanted to push all the nupkgs to blob storage.

Reply:
@johnbeisner wait, we should keep master or we should use release/2.0.0's version in master?

Reply:
History:
We changed 2.0.0 to push ALL the nupkgs to blob storage because the VSO nuget push to VSFEED was timing-out. Then we would push them manually after the build failed. 

Going forward:
Pushing the nupkgs to blob storage is a precautionary step that we would like to keep in 2.0.0 & it should move into master.

Bottom-line: 
no change, merge it